embed.fnc

```diff
@@ -3373,6 +3373,16 @@ AMTdip	|void	|SvREFCNT_inc_void					\
 				|NULLOK SV *sv
 ARdp	|const char *|sv_reftype|NN const SV * const sv 		\
 				|const int ob
+
+Adp	|void	|sv_regex_global_pos_clear				\
+				|NN SV *sv
+Adp	|STRLEN |sv_regex_global_pos_get				\
```

Comment:
_get() should also have flag `R` since calling this function and ignoring the result is pointless.